### DISCUSSION TITLE
Seeking for a mentor

### DISCUSSION THREAD

**Discussion by Real-Chen-Happy** [facebookincubator/velox#9262] [Q&A] [answered]
Hi, all
I am Real and I have some experiences in OLTP systems. My current work is not related to Velox directly. Contributing to Velox is my personal interest because I believe the future of DBMS will be composable, and Velox will definitely play a key role. I would love to contribute my efforts in this area.

I just join Velox community and would love to work on my first issue #3728 . I am seeking for somebody in the community who can help me onboard to the codebase and provide some guidance for my first PR. Thank you for your help!

**Comment by mbasmanova** [comment, marked as answer]
@codyschierbeck Cody, would you or someone from your team be willing to help @Real-Chen-Happy onboard to Velox?

CC: @aditi-pandit

**Comment by aditi-pandit** [comment]
@pramodsatya has offered to help as well. 

Please connect him on Velox or PrestoDB slack.

**Reply by Real-Chen-Happy** [reply]
Thank you so much!

**Comment by codyschierbeck** [comment]
Apologies, I missed this message. 

@Real-Chen-Happy, I'd be glad to help out as well. My experience so far has been 80% in function implementation and about 20% in fuzzer/expression validation. When it comes to questions in those parts of the code base I can certainly provide clarity and take a look at any contributions you make.

I may be able to provide some insight into other parts of the codebase, but not with as much confidence as the other two.

**Reply by Real-Chen-Happy** [reply]
Thank you so much!!

**Reply by codyschierbeck** [reply]
If you have any questions, feel free to @ me on any issues/PR's, or DM me with them and I will try to get back as soon as possible. Velox is an intricate and complicated codebase, so even if it's just "What does this mean?" or "How does this work?" I will try my best to explain.

**Reply by Real-Chen-Happy** [reply]
> If you have any questions, feel free to @ me on any issues/PR's, or DM me with them and I will try to get back as soon as possible. Velox is an intricate and complicated codebase, so even if it's just "What does this mean?" or "How does this work?" I will try my best to explain.

Thank you! That will be very helpful!!

**Reply by codyschierbeck** [reply]
Sorry for the sporadic messages, but one last note. I tried to find you in the Velox slack but nothing came up when I searched "real" in the members list. I believe the Velox team is attending offsites today and tomorrow, so there may be an understandable delay in adding you to the Slack channel. Once you are in though, please send me a DM about #3728. In the meantime, I will ramp myself up on the issue.

I will be away tomorrow and the next day for my birthday, but if you send me a message I will get back to you first thing Monday morning.

**Reply by Real-Chen-Happy** [reply]
Sure! I just emailed the Velox team for joining the slack channel. I will DM you once I joined. Thank you so much for helping me onboard. 

Happy Birthday! 🥳
